benchmarks/query_benchmarks/query_select_related/benchmark.py

```diff
@@ -0,0 +1,14 @@
+from ...utils import bench_setup
+from .models import Book
+
+
+class QuerySelectRelated:
+    def setup(self):
+        bench_setup(migrate=True)
+
+    def teardown(self):
+        Book.objects.all().delete()
+
+    def time_query_select_related(self):
+        for i in range(20):
+            list(Book.objects.select_related("author"))
```

Comment:
Can we have this line repeated 10 times rather than running in a `for` loop?  

With the loop we're then benchmarking cpythons `for` which will have a small(?) overhead.

Reply:
I have changed it